65black/so-lite-react
# #162832591 add search functionality

#### What does this PR do?
This PR adds the ability for a user to search for any question by editing an input element. User does not have to be logged in to search.

#### Description of Task to be completed?
- add tests for new functionality
- separate history from router
- set webpack config to production mode

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Navigate to `localhost:7000/search`, type your search term in the input and hit enter
- If successful, a list of matching results will be rendered
- if unsuccessful, a notice is displayed

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832591

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2019-01-02 at 11 26 00 am" src="https://user-images.githubusercontent.com/36507972/50588275-9e3eda00-0e81-11e9-80f3-580ad1544324.png">
<img width="1440" alt="screenshot 2019-01-02 at 11 26 27 am" src="https://user-images.githubusercontent.com/36507972/50588274-9e3eda00-0e81-11e9-9b67-25d19a48c500.png">

#### Questions:
N/A
